charts/seed-dns/owner/templates/dnsowner.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: dns.gardener.cloud/v1alpha1
+kind: DNSOwner
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  {{- if .Values.ownerID }}
+  ownerId: {{ .Values.ownerID }}
+  {{- end }}
```

Comment:
Field `ownerId` is required.
Either `.Values.ownerID` is set or it makes no sense to deploy a DNSOwner object at all.